calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -25,9 +26,9 @@ def post_usage
             info_we_are_allowed_to_track != "none"
           begin
             HTTPClient.post(ROUTE, info)
-          rescue => _
-            # do nothing
-            # Perhaps we should log?
+          rescue => e
+            message = %Q{ERROR: Could not post usage tracking information:#{$-0}#{e}}
+            Calabash::Cucumber::log_to_file(message)
```

Comment:
Unless the `error` is HTTP related, would it not make sense to try to submit that error to Mixpanel as well?